crates/compiler/tests/compilation_opcodes.rs

```diff
@@ -12,14 +12,14 @@ fn compiles_basic_opcodes() -> miette::Result<()> {
 
     // The number of blocks should be 47 at a _minimum_, as that is the number that
     // appears in the source file. We will definitely should not have allocated more
-    // than 100 total, however.
+    // than 150, however.
     let num_blocks = flo.blocks.iter().count();
     assert!(num_blocks >= 47);
-    assert!(num_blocks < 100);
+    assert!(num_blocks < 250);
```

Comment:
The comments say no more than 150, but the test checks `< 250`. Which is correct?